src/query/api/v1/httpd/handler.go

```diff
@@ -55,10 +54,12 @@ import (
 )
 
 const (
-	healthURL        = "/health"
-	routesURL        = "/routes"
-	engineHeaderName = "M3-Engine"
-	engineURLParam   = "engine"
+	healthURL = "/health"
+	routesURL = "/routes"
+	// EngineHeaderName defines header name which is used to switch between prometheus and m3query engines.
+	EngineHeaderName = "M3-Engine"
+	// EngineURLParam defines query url parameter which is used to switch between prometheus and m3query engines.
```

Comment:
nit: we try to do ~80 chars unless it's super awkward to break up a line; if you're using vscode, this snippet to your `settings.json` will add helpful rulers 👍 

```
    "editor.rulers": [80,120],
    "workbench.colorCustomizations": {
        "editorRuler.foreground": "#888822"
    },
```